lib/terraforming/resource/route53_record.rb

```diff
@@ -66,18 +73,32 @@ def record_sets_of(hosted_zone)
       end
 
       def records
-        hosted_zones.map do |hosted_zone|
+        to_return = hosted_zones.map do |hosted_zone|
           record_sets_of(hosted_zone).map { |record| { record: record, zone_id: zone_id_of(hosted_zone) } }
         end.flatten
+        count = {}
+        dups = to_return.group_by{ |record| module_name_of(record[:record], nil) }.select { |k, v| v.size > 1 }.map(&:first)
```

Comment:
__[rubocop]__
* Space missing to the left of {.
* Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->